docs/dask.md

```diff
@@ -62,7 +64,7 @@ It is generally **recommended** by dask to keep the number of processes low and
 
 It is recommended to use relatively short walltimes for MintPy (the test dataset uses `wall time: 00:30:00` (30 minutes)) to have a higher priority in the job queue, but not too short, or your job may never connect to the dask scheduler and finish its computation (when in doubt, start small and work towards larger wall times as needed). 
 
-Note that the walltime format changes slightly depending on the job scheduler that is in place on your HPC system. For `LSF` schedulers, walltime is accepted and expected to be in `HH:MM` format, while `SLURM` and `PBS` schedulers only accept walltime in `HH:MM:SS` format.
+Note that the walltime format changes slightly depending on the job scheduler that is in place on your HPC system. For `LSF` schedulers, walltime is accepted and expected to be in `HH:MM` format, while `SLURM` and `PBS` schedulers only accept walltime in `HH:MM:SS` format. MintPy will attempt to automatically convert the provided input to the proper format.
```

Comment:
We have two walltime setups: one in the smallbaselineApp.cfg file; one in the YAML file. How do they work together?

The DaskCluster object only format the one in the smallbaselineApp.cfg file, right?
This is the format for the YAML file.

Reply:
I pass inps.walltime to the DaskCluster objet were it is formatted properly. This should handle both the case for specifying it over the command line as well as through smallbaselineApp.cfg

Reply:
So we are 3 ways of specifying walltime. 

I assume our function won't correct for the one typed inside the YAML function, right? Even though the walltime in the YAML file seems never used. If so, this addition of notes does not apply to the YAML file section.

Reply:
There’s no easy way to format the walltime in the yaml file ourselves. If it’s given over the command line or specified in smallbaselineApp.cfg it will be reformatted if necessary.

Reply:
I ended up removing the `--walltime` option completely in favor os having it specified in the `.yaml` files.

Reply:
My only suggestion here is to remove this added note "MintPy will attempt to automatically convert the provided input to the proper format." because it does not apply to the YAML file.

As for the walltime options, it's up to you and @falkamelung.

Reply:
Thing about walltime again, I think we should:

1. keep all the ways (option in template file, option in command line, hardwired in yaml file) available as before. In this way, the values on the template file and/or command line will always overwrite the value in the yaml file, right @Ovec8hkin ?
2. To use the hardwired value on the yaml file, one could assign `none` as a string in the template file or in the command line, then we translate this into a `None` object, and does not assign the walltime in the DaskCluster, so that dask will grab it from the yaml file.

Reply:
I think it’s misleading to allow the user to specify only one of many dask properties over the command line, but not others. Plus, users have to make a dask.yaml with a walltime attribute to use the cluster implementations, so there should never be a need to specify I over the command line.

Reply:
Good point. Then we should also remote the `mintpy.compute.walltime` in the template files, could you do that please?